common/src/main/java/tachyon/security/PlainSaslHelper.java

```diff
@@ -29,10 +40,50 @@
   }
 
   /**
-   * @param name the name of the provider
    * @return true if the provider was registered
    */
   public static boolean isPlainSaslProviderAdded() {
     return Security.getProvider(PlainSaslServerProvider.PROVIDER) != null;
   }
+
+  /**
+   * Get a ClientTransport for Plain
+   * @param username User Name of PlainClient
+   * @param password Password of PlainClient
+   * @param wrappedTransport The original Transport
+   * @return Wrapped Transport with Plain
+   * @throws SaslException
+   */
+  public static TTransport getPlainTransport(String username, String password,
+      TTransport wrappedTransport) throws SaslException {
+    return new TSaslClientTransport("PLAIN", null, null, null, new HashMap<String, String>(),
+        new PlainCallbackHandler(username, password), wrappedTransport);
+  }
+
+  public static class PlainCallbackHandler implements CallbackHandler {
+
+    private final String mUserName;
+    private final String mPassword;
+
+    public PlainCallbackHandler(String mUserName, String mPassword) {
```

Comment:
Constructor parameter `mUserName` and `mPassword` is inconsistent with Tachyon naming convention. `userName` and `password` are good. We only make class members like `mUserName`.